src/content/developers/docs/transactions/index.md

```diff
@@ -138,9 +138,11 @@ Once the transaction has been submitted the following happens:
 2. The transaction is then broadcast to the network and included in a pool with lots of other transactions.
 3. A miner must pick your transaction and include it in a block in order to verify the transaction and consider it "successful".
    - You may end up waiting at this stage if the network is busy and miners aren't able to keep up. Miners will always prioritise transactions with higher `GASPRICE` because they get to keep the fees.
-4. Your transaction will also get a block confirmation number. This is the number of blocks created since the block that your transaction was included in. The higher the number, the greater the certainty that the transaction was processed and recognised by the network. This is because sometimes the block your transaction was included in may not have made it into the chain.
-   - The larger the block confirmation number the more immutable the transaction is. So for higher value transactions, more block confirmations may be desired.
-
+4. Your transaction is assigned a block confirmation number. This number signifies the first block your transaction was included in. Note:
+  - The block confirmation number provides a baseline to estimate the number of blocks created since your transaction's block. The higher the number, the greater the certainty that the transaction was processed and recognised by the network. 
+  - Recent blocks may be re-organised. This gives the impression that the transaction was reversed. The transaction is still valid but may be included in a different block.
+  - The probability of a re-organisation diminishes with every subsequent block mined, i.e. the larger the block confirmation number, the more immutable the transaction is. So, for higher value transactions, more block confirmations may be desired.
```

Comment:
I think we need to clear up exactly what is being defined here.

Originally: "block confirmation number"... was being defined as "number of blocks created since the block that your transaction was included in." I would argue this isn't 100% correct, as it would also include the block the txn was included in. 
Your change to "This number signifies the first block your transaction was included in" actually defines _block height_. Also, this number isn't really "assigned" to the transaction. 

---
Just jotted some other thoughts about the wording in this section:
Once your transaction is mined into a block, that counts as one confirmation. At this point, your transaction has already been recognized by the network and _has been processed_, but there is a small chance the block it was included in isn't permanent, and will need to be processed again and included in another block.